addons/event_sale/views/sale_order_views.xml

```diff
@@ -25,9 +25,8 @@
                 <field
                     name="event_ticket_id"
                     domain="[
-                        ('event_id', '=', event_id),
-                        ('product_id','=',product_id),
-                        '|', ('seats_limited', '=', False), ('seats_available', '>', 0), '|', ('company_id', '=', False), ('company_id', '=', parent.company_id)
+                        ('event_id', '=', event_id), ('product_id','=',product_id),
```

Comment:
Shouldn't we add "name_with_seat_availability" in name_get for event and ticket on SO, that way salespeople can see remaining seats ?

Reply:
They do have it, via the configurator they use when making the order.